Cloud Platform (#64)
OctopusDeploy/Architecture: CloudPlatform/DependencyManagement.md

# Summary #

> Cloud Platform is a relatively small team right now and we relay heavily on other platform teams to help us with work that needs to be done but is not related to the core of Cloud Platform mission.

Reviewer: relay -> rely

Reviewer: "core of Cloud Platform mission." -> Cloud Platform Team's mission.

Author: 👍

> Cloud Platform is a relatively small team right now and we relay heavily on other platform teams to help us with work that needs to be done but is not related to the core of Cloud Platform mission.

Reviewer: ```suggestion
The Cloud Platform Team is relatively small and we rely on other platform teams to help us with work that needs to be done that isn't related to our team mission.
```

# Frontend #

Cloud Portal UI started as a copy of [Octopus Server UI](https://github.com/OctopusDeploy/OctopusDeploy/tree/master/newportal) in 2019. This was done to speed up the development of [Hosted v2](https://docs.google.com/drawings/d/1jl9_g7y_V-PfPrLZY-CdV2cr58mW-A38ieVHn7zFFpg/edit) and to make future updates easier. The first goal has been achieved. The second one not so much and Cloud Portal UI is mostly 2/3 years old at the time of writing this document. 

The UI code is not where Cloud Platform wants to spend most of its time but at the same time it needs to be kept up to date.

> In the future, Team Frontend Platform is planning to develop a platform that other teams can use to build web frontends. We want the future of Cloud Portal UI to be build on top of it so the maintenance cost is reduced to minium.

Reviewer: build -> built
minium -> minimum

> In the future, Team Frontend Platform is planning to develop a platform that other teams can use to build web frontends. We want the future of Cloud Portal UI to be build on top of it so the maintenance cost is reduced to minium.

Reviewer: ```suggestion
In the future, Team Frontend Platform is planning to develop a platform that other teams can use to build web frontends. We want the future of Cloud Portal UI to be built on top of it so the maintenance cost is reduced to a minimum.
```
